apps/web/modules/ee/insights/experience/components/stats.tsx

```diff
@@ -79,20 +87,21 @@ export const ExperiencePageStats = ({ statsFrom, environmentId }: ExperiencePage
             <CardTitle className="text-sm font-medium">{stat.title}</CardTitle>
             <stat.icon className="text-muted-foreground h-4 w-4" />
           </CardHeader>
-          <CardContent className="flex items-center justify-between">
-            <div className="text-2xl font-bold capitalize">
-              {isLoading ? (
-                <div className={cn("h-8 animate-pulse rounded bg-gray-200", stat.width)}></div>
-              ) : (
-                (stat.value ?? "-")
-              )}
-            </div>
+          <CardContent className={cn(isLoading && "animate-pulse")}>
             {stat.key === "sentimentScore" && stats.overallSentiment && (
-              <div>
+              <div className="flex items-center font-medium text-slate-700">
                 {stats.overallSentiment === "positive" ? (
-                  <Badge text="Positive" type="success" size="tiny" />
+                  <TooltipRenderer tooltipContent="Mostly positive">
+                    <SmileIcon className="h-10 w-10" strokeWidth={1.5} />
```

Comment:
<img width="322" alt="Screenshot 2024-10-29 at 06 13 58" src="https://github.com/user-attachments/assets/0d76cf38-6a4f-400e-9353-00734ed303d8">

We lose a lot of information by removing the number. When comparing sentiment between two time periods we don't know if it's going up or down if we just see the same smile/category. Please discuss with chatGPT what the best scientific approach is to display the development of sentiment over time in a dashboard.